packages/grid/src/interfaces.ts

```diff
@@ -3,10 +3,30 @@ import { ReactElement, CSSProperties, RefObject } from 'react';
 import { ITheme } from '@xcritical/theme';
 
 import { GridPositions, GridSort } from './consts';
+import { Alignment } from 'react-virtualized';
 
 
 type CellContent = string | number | ReactElement | any;
 
+export type InternalGridProps = {
+  'aria-label'?: string,
+  'aria-readonly'?: boolean,
+  className?: string,
+  containerProps?: object,
+  containerRole?: string,
+  containerStyle?: React.CSSProperties,
```

Comment:
CSSProperties is already imported into the file, no need to write React before type CSSProperties